assets/dev/scss/editor/preview/preview.scss

```
@@ -524,6 +524,10 @@ html.elementor-html {
 		filter: opacity(0.4) saturate(0);
 	}
 
+	&:not(:has(> .elementor-widget-container)) {
```

I have a general question: is is possible to avoid these style duplications, like this: 

```suggestion
	> .elementor-widget-container, > .elementor-widget-wrap,
	&:not(:has(> .elementor-widget-container)) {
```